LDMP/plugin.py

```diff
@@ -153,59 +168,70 @@ def add_action(
     def initGui(self):
         self.initProcessing()
 
-        """Create the menu entries and toolbar icons inside the QGIS GUI."""
-        self.add_action(
-            ':/plugins/LDMP/icons/wrench.svg',
-            text=self.tr(u'Settings'),
-            callback=self.run_settings,
-            parent=self.iface.mainWindow(),
-            status_tip=self.tr('Trends.Earth Settings'))
-
+        """Create Main manu icon and plugins menu entries."""
         self.add_action(
-            ':/plugins/LDMP/icons/calculator.svg',
-            text=self.tr(u'Calculate indicators'),
-            callback=self.run_calculate,
+            ':/plugins/LDMP/icons/trends_earth_logo_square_32x32.ico',
+            text=self.tr(u'Trend.Earth'),
```

Comment:
Maybe a typo, but please use `Trends.Earth` here, as it is the proper name of the plugin